comic_recommender.py

```
@@ -0,0 +1,15 @@
+from typing import List, Dict, Tuple, Optional
```

Flake8 (7.2.0)

[error] 'typing.Tuple' imported but unused

(F401)